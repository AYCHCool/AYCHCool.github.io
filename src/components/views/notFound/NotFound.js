import React from 'react';

const NotFound = () => (
  <div>
    <h3>MAZUS TEAM is still building out this app, your page cannot be found at the moment</h3>
  </div>
);
export default NotFound;
