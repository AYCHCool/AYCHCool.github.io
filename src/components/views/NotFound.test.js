import React from 'react';
import NotFound from './NotFound';

it('renders correctly', () => {
  const wrapper = mount(<NotFound />);
  expect(toJson(wrapper)).toMatchSnapshot();
});
