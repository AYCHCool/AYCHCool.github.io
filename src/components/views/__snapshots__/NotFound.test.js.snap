// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<NotFound>
  <div>
    <h3>
      MAZUS TEAM is still building out this app, your page cannot be found at the moment
    </h3>
  </div>
</NotFound>
`;
